crates/pallets/starknet/src/lib.rs

```diff
@@ -190,6 +189,15 @@ pub mod pallet {
         fn offchain_worker(n: T::BlockNumber) {
             log!(info, "Running offchain worker at block {:?}.", n);
 
+            let addr = StorageValueRef::persistent(b"starknet::seq_addr");
+
+            if let Ok(Some(address)) = addr.get::<[u8; 32]>() {
+                log::info!("Current set sequencer address: {:?}", address);
+                ()
+            } else {
+                log::info!("Sequencer address not set");
+            }
+
```

Comment:
What is the purpose of this? You are reading but not doing anything

Reply:
Fair enough, especially with the idea to provide the address in the future with a cli flag there is no need for the logging, will remove it